mautic/mautic
# imageDefaultWidth set to 0 which means any default width is not set

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Not really
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2170#issuecomment-250178805
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Froala's default behavior is that every inserted image has width of 300px. @chandon suggests to turn this off. This PR does that. According to my tests the image now gets the full width of the block element which surrounds the image.

#### Steps to test this PR:
1. Apply this PR.
2. Refresh the prod assets or use the dev mode.
3. Try to insert an image again. The width should be either maximal image width or width of the surrounding element.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Try to insert image to any editor area.
2. Check the image width. It should be exactly 300px.
